shared/actions/json/settings.json

```diff
@@ -48,7 +48,7 @@
     "editPhone": {
       "phone": "string",
       "delete?": "boolean",
-      "toggleSearchable?": "boolean",
+      "setSearchable?": "boolean",
       "verify?": "boolean"
```

Comment:
should we get rid of `verify`? Looks like the saga doesn't use it anymore.

Reply:
How does verification work then?

Reply:
It uses `resendVerificationForPhoneNumber`

Reply:
And `addPhoneNumber` automatically sends a verification text